tests/test_cases/input/escline_alone.kdl

```diff
@@ -0,0 +1 @@
+\
```

Comment:
Now I'm confused: is this actually allowed? I think it should be (just allow all whitespace to be escaped even if it's meaning less) but right now I'm not sure what the spec actually says right now